src/PFXImportPowershell/README.md

```diff
@@ -14,6 +14,10 @@ This project consists of helper Powershell Commandlets for importing PFX certifi
 	# 1. Setup Key (if you don't have a dedicated provider, you can use "Microsoft Software Key Storage Provider")
 	Add-IntuneKspKey "<ProviderName>" "<KeyAlgoritm>" "<KeyName>"
 	
+## Authenticate to Intune
+    # 1. Authenticate as the account administrator (using the admin UPN) to Intune.
+    $authResult = Get-IntuneAuthenticationToken -AdminUserName "<Admin-UPN>"
```

Comment:
Do we want to mention the optional password?

Reply:
Done.